casper/src/test/scala/coop/rchain/casper/MaybeCellSpec.scala

```diff
@@ -0,0 +1,39 @@
+package coop.rchain.casper
+import coop.rchain.casper.protocol.ApprovedBlock
+import coop.rchain.shared.MaybeCell
+import monix.eval.Task
+import monix.execution.schedulers.TestScheduler
+import org.scalatest.{FunSpec, Matchers}
+
+import scala.util.Success
+
+class MaybeCellSpec extends FunSpec with Matchers {
+
+  describe("MaybeCell") {
+    it("get should return ApprovedBlock on get when available") {
```

Comment:
would be nice to have this test emulate two concurrent threads (two for-comprehensions)

In the same spirit, it'd be good to check that if I complete a block in another thread after the `getOptional`'s timeout, it does not affect the result of the `getOptional`.

Reply:
There's no `getOptional` with timeout anymore.